angr/procedures/libc/strtol.py

```diff
@@ -131,7 +133,7 @@ def _string_to_int(s, state, region, base, signed, read_length=None):
         return expression, result, num_bytes
 
     @staticmethod
-    def _char_to_val(char, base):
+    def _char_to_val(char, state, base):
         """
```

Comment:
What's the rationale behind this change? `claripy.BVV` is the same as `state.se.BVV`. Also, `state.se` is deprecated anyway (you should use `state.solver`).